features/importers/myhelsinki_places/importer.py

```diff
@@ -136,6 +154,40 @@ def import_feature_contact_info(self, feature: Feature, address: dict):
                 },
             )
 
+    def import_opening_hours(self, feature: Feature, opening_hours: dict):
+        """Imports opening hours for the given feature."""
+
+        def has_data(hours):
+            return bool(hours["opens"] or hours["closes"] or hours["all_day"])
+
+        filtered_hours = filter(has_data, opening_hours["hours"])
+        hours = list(filtered_hours)
+
+        if opening_hours["remark"] or hours:
+            ohp, created = OpeningHoursPeriod.objects.get_or_create(
+                feature=feature, defaults={"remark": opening_hours["remark"]}
```

Comment:
isn't it possible that this would return multiple objects? `feature` is a `FK` field 🤔

Reply:
Theoretically that's possible yes and our DB design does indeed support that. E.g. a place could have two or more different opening hours (for restaurant and sauna&swimming pool sections). Or we might have opening hours that are valid on 1.6.2020-31.8.2020 and on 1.9.2020-31.5.2021.

For MyHelsinki Places API the situation of course is that it provides only one set of opening hours. I could however try to handle that situation here so that it wouldn't cause an `Exception`. :thinking: